web/assets/js/components/gameboard.ts

```diff
@@ -269,23 +286,27 @@ const Gameboard = class extends Component {
 
       // Fetch the move history to check for a capture
       const moveHistory = this.chess.history({ verbose: true });
-      if (moveHistory.length == 0) {
-        await this.engine();
+      console.log(moveHistory);
 
-        return;
+      //TODO: not sure we need it
```

Comment:
I thought so too, when I initially added this check;

It doesn't make sense why we'd need to check, if the game _has been confirmed to be changed_, so the `moveHistory` is almost certainly not empty. I was running into an issue where it actually was empty, and the game has changed 🤷‍♂️ 

I think you've fixed it though with `const currentFen = new Chess(gameState.position.fen).fen();`